core/lib/refinery/crud.rb

```diff
@@ -83,6 +83,12 @@ def create
           def edit
             # object gets found by find_#{singular_name} function
           end
+          
+          def duplicate
+            @#{singular_name} = @#{singular_name}.dup
+            @#{singular_name}.title << ' Copy'
```

Comment:
perhaps `' Copy'` should be a method return so that it's easy to change.. like:

```ruby
def duplicate
  @#{singular_name} = @#{singular_name}.dup
  @#{singular_name}.title << duplicate_suffix
  render 'edit'
end

private

def duplicate_suffix
  ' Copy'
end
```

Another idea is we could get it from a locale file so that it's not always English